### PR TITLE
Add Numbering for Ingredients in Fridge

### PR DESCRIPTION
**Pull request by oliviajohansen**

### CODE REVIEW

**Review by caitlinjee — APPROVED** [review]
apart from some comments, lgtm

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,19 @@ layout: page
 title: User Guide
 ---
 
-Wishful Shrinking is a **desktop app for managing your diet, keeping track of your on-hand ingredients, recipes, as well as the food you’ve eaten (along with their calories)**. It is optimized for fast and efficient typist as it uses a **Command Line Interface (CLI)** with the added beauty of a Graphical User Interface (GUI).
+Welcome to the Wishful Shrinking User Guide! This user guide provides in-depth documentation on the installation process, step-by-step instructions for each feature and troubleshooting recommendations. Wishful Shrinking is available for the Linux, Unix, Windows and Mac OS operating systems.
+
+
+## Introducing Wishful Shrinking
+Wishful Shrinking your desktop diet manager. It is an app that helps you manage your on-hand ingredients, organise personal recipes and track your diet. Wishful Shrinking facilitates a healthier diet in three main ways:
```
Comment (caitlinjee) [inline docs/UserGuide.md]: is it wishful shrinking is your...

File: docs/UserGuide.md [hunk 2]
```diff
@@ -174,6 +216,33 @@ Examples:
 * `list` followed by `deleteF 2` deletes the 2nd ingredient in the fridge.
 * `searchF peanut` followed by `deleteF 1` deletes the 1st ingredient in the results of the `search` command.
 
+### Edit an ingredient: `editF`
+
+Edits the specified ingredient from Ingredient Collection.
```
Comment (caitlinjee) [inline docs/UserGuide.md]: should it be ingredient list?

File: docs/UserGuide.md (hunk 1 above)
Comment (oliviajohansen) [reply docs/UserGuide.md]: Wishful Shrinkng- your desktop diet manager. Is this clearer?

File: docs/UserGuide.md (hunk 2 above)
Comment (oliviajohansen) [reply docs/UserGuide.md]: ok

**Review by daongochieu2810 — APPROVED** [review]

**Review by TianYong-Goh — APPROVED** [review]
lgtm

**Review by jiaqi20 — APPROVED** [review]
lgtm

File: docs/UserGuide.md (hunk 1 above)
Comment (caitlinjee) [reply docs/UserGuide.md]: okok can!

**Pull request merged**